### PR TITLE
Fix GitHub issue 571

### PR DESCRIPTION
**Pull request by pengz1**
This PR is to fix git hub issue 571: get_driveid.js. is not creating the virtualDisk attribute.
Issue 571 is caused by  "vd" attribute missing for JBOD disks, this PR will default set "vd" to an empty string for all disks.

### CODE REVIEW

**Comment by pengz1** [comment]
@anhou @ytjohn @yyscamper

**Comment by pengz1** [comment]
test this please

File: data/templates/get_driveid.js [hunk 1]
```diff
@@ -189,14 +189,12 @@ function parseVdInfo(pathList) {
     });
 
     return pciLines.map(function(line) {
-        var scsiId = line[1].slice(line[1].lastIndexOf('-') + 1), vdStr;
+        var scsiId = line[1].slice(line[1].lastIndexOf('-') + 1),
```
Comment (yyscamper) [inline data/templates/get_driveid.js]: Could you please paste some example line here, so that it helps to understand your code.

**Comment by pengz1** [comment]
test this please

**Comment by iceiilin** [comment]
:+1:

**Pull request merged**